src/main/kotlin/com/terraformation/backend/accelerator/db/SubmissionStore.kt

```diff
@@ -35,27 +35,35 @@ class SubmissionStore(
       deliverableId: DeliverableId,
       projectId: ProjectId,
       status: SubmissionStatus = SubmissionStatus.NotSubmitted,
-  ) {
+  ): SubmissionId {
     requirePermissions { createSubmission(projectId) }
 
+    if (status != SubmissionStatus.NotSubmitted && status != SubmissionStatus.Completed) {
```

Comment:
I added a checker to make sure the user is not creating a submission in a status that is supposed to be handled by an internal admin.